### ISSUE TITLE
[AXE-CORE]: Change some of the H2s to H3s

### ISSUE THREAD

**Issue by TiffanyPender** [department-of-veterans-affairs/va.gov-cms#9722]
- [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

## Feedback framework

- **⚠️ Should** (best practice)

## Definition of done

1. Review and acknowledge feedback.
2. Fix and/or document decisions made.
3. Acceptance criteria listed below met.
4. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

---

## User Story or Problem Statement

As a user who navigates the content of a web page by the headings mapping of the page, I expect headings to be organized in a way that helps me understand the structure and meaning of the page.

## Details

There is currently a content area on the home page that includes the heading level 2 (`h2`) text of "Learn about and apply for VA benefits and health care". All of the sub-headings underneath of this section are also using heading level 2 elements. This can be confusing to screenreader users and users who navigate with the headings mapping of the page who may be expecting these to be sub-categories of the previous `h2`. It is recommended that these sub-categories use the `h3` element.

## Acceptance Criteria

- [ ] Sub-categories listed after the `h2` text "Learn about and apply for VA benefits and health care" are `h3` elements. Sub-categories include:
  - [ ] *Service member benefits*
  - [ ] *Family member benefits*
  - [ ] *Burials and memorials*
  - [ ] *Careers and employment*
  - [ ] *Housing assistance*
  - [ ] *Pension*
  - [ ] *Life insurance*
  - [ ] *Education and training*
  - [ ] *Records*
  - [ ] *Health care*
  - [ ] *Disability*

## Environment

* macOS Monterey, `<Version 12.4>`
* Chrome, `<Version 102.0.5005.61>`
* VoiceOver
* Staging

## Steps to Recreate

1. Enter `https://staging.va.gov/homepage-test/` in browser address bar
2. Start screenreading device listed in Environment and listen to the headings of the sub-categories that come after the "Learn about and apply for VA benefits and health care" `h2` text *OR* 
3. Search the `h2` headings of the page using developer tools or a headings mapping tool for the sub-category heading elements that appear under the "Learn about and apply for VA benefits and health care" `h2` text (`h2#learn-about-and-apply-for-va-b`)

## Proposed Solution (if known)

Update the "Learn about and apply for VA benefits and health care" sub-categories as `h3` elements instead of `h2` elements.

## WCAG or Vendor Guidance (optional)

* [W3C Headings](https://www.w3.org/WAI/tutorials/page-structure/headings/)
* [Accessible heading structure](https://www.a11yproject.com/posts/how-to-accessible-heading-structure/)

## Screenshots or Trace Logs

None

**Comment by jcklpe** [comment]
Let me see if I understand everything right:

*   First picture is how it is currently
    *   It's an h1 but styled like an h2
*   Second picture is the redesign
    *   It's an h2 but semantically should be an h1

Solution to me seems to be: make it an h1, style it like an h1. Form follows function and all that. Can't imagine that a h1styling would really be that big a deal, and semantics take precedence IMO. When you start changing the design to be different from the semantics it indicates to me a weakness in the design to begin with.

**Comment by jcklpe** [comment]
The solution proposed seems good to me. It's good to follow proper semantics.   
  
Is there a question raised here about if we should change the styling to h3s also? I think it's best for semantics and styling to be in sync, but perhaps this is not something we can do at this time.

**Comment by jilladams** [comment]
If you have questions about where the /homepage-test code lives, @allisonlu built it and can help

**Comment by wesrowe** [comment]
> It would be good to get feedback from the designer of this area as well.
@TiffanyPender, to close the loop on this, the intent on the visual design was to keep these particular elements exactly as they are on the real home page in Prod today. So the "designer" is the status quo. Hopefully that helps clarify the intent here.

**Comment by mmiddaugh** [comment]
@tiffany

**Comment by mmiddaugh** [comment]
@TiffanyPender, will you please validate this change resolves the defect, when you are able? Thank you!

**Comment by TiffanyPender** [comment]
Looks good! @mmiddaugh

<img width="1394" alt="VA home page on staging - Verified heading levels" src="https://user-images.githubusercontent.com/25351510/181830524-9f73e49e-4281-4e07-9512-dc35d10a3c82.png">
